cypress/e2e/pages/homePage/index.cy.js

```diff
@@ -1,33 +1,75 @@
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
 import { HOME_PAGE } from '#app/routes/utils/pageTypes';
-import getUrls from './urls';
-import getAppEnv from '../../../support/helpers/getAppEnv';
-import visitPage from '../../../support/helpers/visitPage';
-import crossPlatformTests from './tests';
 import canonicalTests from './testsForCanonicalOnly';
 
-const pageType = HOME_PAGE;
+const tests = [canonicalTests];
 
-const environment = getAppEnv();
-const urls = getUrls();
+const testSuites = [
```

Comment:
Do we want/need to reduce this list - I see we only have persian local - but we have arabic in all environments. 

Similarly, we have pidgin, but only on local + test, whereas we have kyrgyz, portuguese, polska in all environments etc. 

It's fine to keep them - there are no smoke tests, but the tests for the home page are pretty quick - we only run tests which we couldn't run in Integration tests.